components/Profile.jsx

```diff
@@ -1,7 +1,40 @@
 import React from "react";
 
-const Profile = () => {
-  return <div>Profile</div>;
+const Profile = ({ name, major, classOf, netId, email, points }) => {
+  return (
+    <div className="grid grid-cols-2 w-[450px] p-[142px]">
+      <div>
+        <h1 className="text-[36px] h-fit w-fit font-bold">name:</h1>
```

Comment:
use p tags

Reply:
I'm a little confused, in another commit I saw that we want to be using react grid elements rather than css grid styles. Not sure if that still applies here but might also be something to look into.

I also would use a p tag at least for where we are injecting values using props.

Reply:
Yea we want to be using Bootstrap grid